### PR TITLE
Feat/comments

### PR DESCRIPTION
**Pull request by thipereira02**

### CODE REVIEW

File: src/components/Comments.js [hunk 1]
```diff
@@ -0,0 +1,190 @@
+import styled from "styled-components";
+import { FiSend } from "react-icons/fi";
+import { useContext } from "react";
+import UserContext from "../contexts/UserContext";
+import { postNewComment } from "../service/api.service";
+import { Link } from "react-router-dom";
+
+export default function Comments({ postComments, userId, postId, myComment, setMyComment }) {
+    const {userData, onChangePost, setOnChangePost, followsId } = useContext(UserContext);
+    console.log(postComments)
```
Comment (calilrenner) [inline src/components/Comments.js]: Thiago, please, could you remove this console.log?

File: src/pages/acessControl/Login.js [hunk 2]
```diff
@@ -18,9 +18,7 @@ export default function Login() {
     if (userDataJSON) {
       login(JSON.parse(userDataJSON));
     }
-  }, 
-  []// eslint-disable-line react-hooks/exhaustive-deps
-  );
+  }, []); // eslint-disable-line react-hooks/exhaustive-deps
```
Comment (calilrenner) [inline src/pages/acessControl/Login.js]: Thiago, please, could you remove this comment?

**Review by calilrenner — APPROVED** [review]
In general, it looks all right. All requirements seems to be running on Vercel.

**Review by sidneyouteiro — CHANGES_REQUESTED** [review]
CSS isn't prepared for big comments that don't have whitespace, I think [this](https://css-tricks.com/snippets/css/prevent-long-urls-from-breaking-out-of-container/) might help
image

**Review by sidneyouteiro — APPROVED** [review]

**Review by sidneyouteiro — APPROVED** [review]

**Pull request merged**